backstop/tests/radio.html

```diff
@@ -106,7 +106,7 @@ <h2>Default</h2>
                 <circle
                   cx="8"
                   cy="8"
-                  r="6"
+                  r="4"
```

Comment:
Do we really need this radius change?

Reply:
Our old radio dots had a 16px canvas, but the dots were actually 12px.  Then we displayed that 16px artboard at 12px which decreased the dot size to 9px.

With this new method, the 16px canvas is displayed at 100% (which happens to be 16px), so now whatever size we make the radius, it will actually display at that size.  I thought about using `r="4.5"` so that our test images wouldn't change, but decided to go with `r="4"` so that the dot was an even number (8px) instead of odd (9px).